app/views/govuk_publishing_components/components/_contextual_sidebar.html.erb

```diff
@@ -27,4 +27,8 @@
   <% if navigation.show_brexit_cta? %>
     <%= render 'govuk_publishing_components/components/contextual_sidebar/brexit_cta', content_item: content_item %>
   <% end %>
+
+  <% if navigation.show_ukraine_cta? %>
```

Comment:
If this goes live before we have had sign off that we can remove the brexit cta, do we know that this is the order they want the cta's to be? Sort of feels like the Ukraine CTA should come higher up the page but I might be out of the loop.

Reply:
Makes sense – product confirmed the suggested order so I've updated.